packages/office-ui-fabric-react/src/components/Calendar/Calendar.tsx

```diff
@@ -115,6 +115,7 @@ export class Calendar extends BaseComponent<ICalendarProps, ICalendarState> impl
                   onNavigateDate={ this._onNavigateDate } /> }
 
                 { showGoToToday && <span
+                  role='button'
```

Comment:
You need to add space as an activation key, too. Do the action shift focus to the current date?